canonical/charmhub.io
# Fix filtering

## Done
- fix filtering based on categories
- filter out archived posts
## How to QA
- Go to https://charmhub-io-1644.demos.haus/docs?{search-term} and ensure all `category_id` in the topics are either doc(22) or its sub-categories (34, 47, 43)
- Ensure that no topic has status `archived` to be true
## Issue / Card
Fixes #

## Screenshots
